modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -297,6 +297,14 @@ export default {
         withPublished: 1
       };
 
+      const type = this.relationshipField?.withType;
+      const isPage = apos.modules['@apostrophecms/page'].validPageTypes
+        .includes(type);
+
+      if (isPage) {
+        options.type = type;
+      }
+
```

Comment:
this little block makes the magic happen

Reply:
Beauty